provider/aws.go

```diff
@@ -230,6 +230,10 @@ func (p *AWSProvider) Records() (endpoints []*endpoint.Endpoint, _ error) {
 		return nil, err
 	}
 
+	return p.records(zones)
+}
+
+func (p *AWSProvider) records(zones map[string]*route53.HostedZone) (endpoints []*endpoint.Endpoint, _ error) {
```

Comment:
This should be `func (p *AWSProvider) records(zones map[string]*route53.HostedZone) ([]*endpoint.Endpoint, error)` to stick with the other functions and methods

Reply:
I will fix it but its how Records() was stamped.